models/src/main/java/com/vimeo/networking2/TeamPermission.kt

```diff
@@ -0,0 +1,26 @@
+package com.vimeo.networking2
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Represents the current permission relationship a [teamEntity] has with a resource. This class expresses any current
+ * way the [TeamEntity] is associated with the resource through [currentPermissions].  This class expresses any way the
+ * [TeamEntity] can have it's permission relationship to a resource edited via the set of potential permissions listed
```

Comment:
`it's` -> `its`. Can we add parameter documentation for the properties of this class?

Reply:
The parameters are explained inline by being mentioned in the description. I'll move them to params.